hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/security/token/DistributedSQLCounter.java

```diff
@@ -50,7 +50,9 @@ public DistributedSQLCounter(String field, String table,
 
   /**
    * Obtains the value of the counter.
+   *
    * @return counter value.
+   * @throws SQLException if SQL connection errors occur.
```

Comment:
`@throws SQLException if SQL errors occur` is a tautology.

`@throws SQLException if querying the database fails` may be slightly better, but it also doesn't say much.

This is a minor issue though. The forced annotations make the javadocs kinda verbose so it is okay to use the form you have.